src/opensles/AudioStreamOpenSLES.cpp

```diff
@@ -383,6 +383,9 @@ Result AudioStreamOpenSLES::close_l() {
     EngineOpenSLES::getInstance().close();
 
     setState(StreamState::Closed);
+
+    closePerformanceHint();
```

Comment:
Where is this defined for OpenSL ES? I only see a definition in AAudio

Reply:
I removed the call. It is not supported in OpenSL ES.